task2.js

```diff
@@ -0,0 +1,23 @@
+//https://edabit.com/challenge/FzDAgSR84zeyRa278
+
+// Extract City Facts
+
+const city = [
+  {
+    name: "Paris",
+    population: "2,140,526",
+    continent: "Europe"
+  },
+  {
+    name: "Tokyo",
+    population: "13,929,286",
+    continent: "Asia"
+  }
+];
+
+function cityFacts() {
+  console.log(city[0].name + ' has a population of ' + city[0].population + ' and is statuated in ' + city[0].continent);
+  console.log(city[1].name + ' has a population of ' + city[1].population + ' and is statuated in ' + city[1].continent);
+}
```

Comment:
Pani Ewo, ta funkcja ma przyjmować obiekt i zwracać `string` w formacie np. 
`Paris has a population of 2,140,526 and is situated in Europe`

I prosze do tego zadania, i do pierwszego wkleić całe polecenie z przykładami testowymi.